playbook/app/pb_kits/playbook/pb_dialog/_dialog.scss

```diff
@@ -0,0 +1,111 @@
+@import "../tokens/positioning";
+@import "../tokens/colors";
+@import "../pb_card/card_mixin";
+@import "../tokens/shadows";
+
+@keyframes modalFadeIn {
+	from {
+		transform: translate3d(0, -100%, 0);
+		opacity: 0;
+	}
+	to {
+		transform: translate3d(0, 0, 0);
+		opacity: 1;
+	}
+}
+
+@keyframes modalFadeOut {
+	from {
+		transform: translate3d(0, 0, 0);
+		opacity: 1;
+	}
+	to {
+		transform: translate3d(0, -50%, 0);
+		opacity: 0;
+	}
+}
+
+@keyframes overlayFade {
+	from {
+		opacity: 0;
+		transform: translateY(0);
+	}
+	to {
+		opacity: 1;
+		transform: translateY(0);
+	}
+}
+
+@keyframes overlayFadeOut {
+	from {
+		opacity: 1;
+	}
+	to {
+		opacity: 0;
+	}
+}
+
+.dialog {
+	@include pb_card;
+	max-height: calc(100vh - 64px);
+	max-width: calc(100vw - 64px);
+	overflow: scroll;
+	animation-name: modalFadeIn;
+	animation-duration: 0.2s;
+	outline: none;
+	animation-timing-function: cubic-bezier(0.7,0,0.3,1);
```

Comment:
Any chance to use our variables? https://github.com/powerhome/playbook/blob/master/playbook/app/pb_kits/playbook/tokens/_animation-curves.scss